scripts/setup-k8s-master.sh.in

```diff
@@ -55,41 +55,61 @@ kubeadm reset --force
 
 cat >/tmp/kubeadm.yaml <<EOF
 ---
-apiVersion: kubeadm.k8s.io/v1alpha2
-kind: MasterConfiguration
-kubernetesVersion: "$(cat /etc/kubernetes_community_ami_version)"
-api:
-  controlPlaneEndpoint: "${LB_DNS}:443"
-apiServerCertSANs:
-- "${LB_IPV4}"
-apiServerExtraArgs:
-  cloud-provider: "aws"
-nodeRegistration:
-  name: "${HOSTNAME}"
-  kubeletExtraArgs:
-    cloud-provider: "aws"
+apiVersion: kubeadm.k8s.io/v1beta1
 bootstrapTokens:
 - groups:
-  - "system:bootstrappers:kubeadm:default-node-token"
+  - system:bootstrappers:kubeadm:default-node-token
   token: "{{ClusterToken}}"
-  ttl: "0s"
+  ttl: 0s
```

Comment:
So I did the config update by hand and apparently left out a few quotes; not sure if that matters.

Reply:
I think this is ok since YAML *should* interpret it as a string. I believe the quotes were there for extra protection. Likely not necessary in these cases

Reply:
It might be worth it to add back the quotes around strings containing a `:` since that can potentially confuse yaml parsers .